style.css

```diff
@@ -55,34 +54,34 @@ li {
   padding: 10px;
 }
 
-
-
 /* Nav CSS Start */
-.flex-nav ul{
+.flex-nav ul {
   display: flex;
   margin: 0;
   border: 1px solid red;
 }
+
 .flex-items {
   font-weight: 500;
   font-size: 12px;
   line-height: 20px;
-  margin: 0px 5px;
+  margin: 0 5px;
   text-align: center;
 }
+
 .nav-name {
   flex: 1;
   align-self: flex-start;
   border: 1px solid red;
 }
+
 #nav-icon {
   width: 50%;
   align-self: flex-end;
-  border: 1px solid red; 
+  border: 1px solid red;
 }
```

Comment:
- [ ] Please follow [the best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) documents. In this particular case, don't use the id selector for generic components, instead use the attribute "class" to refer to them. This will help you keep a more organized, cleaner, and [DRY code](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Reply:
True, I end up with the same id in <section.social>. All issues were addressed, thank you for the code review.